net-engine/job-skin-builder
# [PLAN-1291] Add Open Graph meta tags

# Related issue(s)



https://netengine-trial.atlassian.net/browse/PLAN-1291?atlOrigin=eyJpIjoiNDUzMDI4YTg0MGY3NGU1YzlhNGU3YmQ2OTU1MzNmYjAiLCJwIjoiaiJ9

# Description

Adds open graph meta tags to job skins.

This will be manually deployed after the 28/06/2022 release to coincide with the back end changes that allow further job information to be added into these OG meta tags.

# Type of Change


- [ ] Request (code for a specific purpose that will not add a feature. e.g. one off rake task)
- [ ] Maintenance task (rake tasks or changes that do not change logic tha will affect the application behaviour)

# How to test / reproduce

yarn start, create a job skin with the social media share preview section filled out. Make sure the og meta tags are generated correctly.

## Assignee

Phil

# Screenshots

<img width="1207" alt="Screen Shot 2022-06-27 at 11 02 25 am" src="https://user-images.githubusercontent.com/92001562/175842392-036ee6e7-652d-4f1d-8b6d-971e75e7721a.png">

# Checklist

- [x] The implementation has been tested locally
- [x] The author of this PR is the Assignee
- [x] There are no typos, spelling, or grammatical errors in end-user facing text
